### PR TITLE
Add VoidZero to members

### PR DESCRIPTION
**Pull request by yyx990803**
- Blog post: https://voidzero.dev/posts/oss-pledge
- JSON: https://voidzero.dev/oss-pledge.json
- Square icon (png): https://avatars.githubusercontent.com/u/149750581?s=400&v=4
- Wordmark (svg): https://voidzero.dev/logo.svg
- Brand assets in more formats: https://voidzero.dev/branding

### CODE REVIEW

**Comment by vladh** [comment]
Thanks so much for joining @yyx990803, it's amazing to have you on board! 🎉 

Everything looks perfect to me, especially the blog post. I'll just wait for a second review.

**Review by vladh — APPROVED** [review]

**Comment by chadwhitacre** [comment]
Welcome aboard! :)

**Pull request merged**
